tests/parser/tests/cases/should_pass/nested_re_assignment/case.hash

```diff
@@ -0,0 +1,2 @@
+// This technically should parse, but it wouldn't make sense since the '+=' operator has a void return type
```

Comment:
Is that a part of the trait definition?

Reply:
No i don't think it should be, ill remove this comment.